elastic/tests/test_metrics.py

```diff
@@ -41,7 +41,7 @@
         pytest.param([5, 0, 0], True, 254, id='jmx-rate-v5'),
         pytest.param([6, 3, 0], True, 254, id='jmx-rate-v6.3.0'),
         pytest.param([7, 2, 0], True, 248, id='jmx-rate-v7.2.0'),
-        pytest.param([7, 9, 0], True, 251, id='jmx-rate-v7.9.0'),
+        pytest.param([7, 9, 0], True, 261, id='jmx-rate-v7.9.0'),
```

Comment:
Should we also add `7.10.X` to all these `parametrize` tests?

Reply:
tbh I don't understand the purpose of these tests, they were pretty annoying to have to update.

These tests also test implementation details, not behavior. I'm tempted to delete the existing ones instead of adding new ones.